server/main.lua

```diff
@@ -49,6 +49,12 @@ RegisterNetEvent('qb-vehiclekeys:server:setVehLockState', function(vehNetId, sta
     SetVehicleDoorsLocked(NetworkGetEntityFromNetworkId(vehNetId), state)
 end)
 
+RegisterServerEvent("qb-vehiclekeys:server:synckeys", function(KeyList)
+    if not Config.SaveInDB then return end
+    local Player = QBCore.Functions.GetPlayer(source)
+    Player.Functions.SetMetaData("VehKeys", KeyList)
+end)
+
 QBCore.Functions.CreateCallback('qb-vehiclekeys:server:GetVehicleKeys', function(source, cb)
     local Player = QBCore.Functions.GetPlayer(source)
     if not Player then return end
```

Comment:
```suggestion
    if not Player then
        cb({})
        return
    end
```

This could potentially cause issues, as the client is waiting for a respone it will never receive.

I guess we should have an explicit callback saying the player isnt loaded, though, perhaps?
Although, this callback shouldnt be called if the player aint loaded in the first place (hopefully)...?